tests/pxcentral/pxcentral_test.go

```diff
@@ -0,0 +1,173 @@
+				if i == 2 {
+					break
+				}
+				err := Inst().S.AddLabelOnNode(node, "px/ls", "true")
+				Expect(err).NotTo(HaveOccurred())
+			}
+
+			err := Inst().S.AddTasks(context, lsOptions)
+			Expect(err).NotTo(HaveOccurred())
+
+			ValidateContext(context)
+			logrus.Infof("Successfully validated specs for px-license-server")
+		})
+
+		Step("Install px-monitor then validate", func() {
+			var endpoint, oidcSecret string
+			Step("Getting px-backup UI endpoint IP:PORT", func() {
+				endpointIP := node.GetNodes()[0].GetMgmtIp()
+
+				serviceObj, err := core.Instance().GetService("px-backup-ui", context.GetID())
+				Expect(err).NotTo(HaveOccurred())
+				endpointPort := serviceObj.Spec.Ports[0].NodePort
+
+				endpoint = fmt.Sprintf("%s:%v", endpointIP, endpointPort)
+				logrus.Infof("Got px-backup-ui endpoint: %s", endpoint)
+			})
+
+			Step("Getting OIDC client secret", func() {
+				secretObj, err := core.Instance().GetSecret("pxc-backup-secret", context.GetID())
+				Expect(err).NotTo(HaveOccurred())
+
+				secretData, exist := secretObj.Data["OIDC_CLIENT_SECRET"]
+				Expect(exist).To(Equal(true))
+				oidcSecret = base64.StdEncoding.EncodeToString(secretData)
+				logrus.Infof("Got OIDC client secret: %s", oidcSecret)
+			})
+
+			Step("Adding values to helm values ConfigMap", func() {
+				configMap, err := core.Instance().GetConfigMap(Inst().HelmValuesConfigMap, "default")
+				Expect(err).NotTo(HaveOccurred())
+
+				monitorValues, exist := configMap.Data[monitorApp]
+				Expect(exist).To(Equal(true))
+
+				configMap.Data[monitorApp] = fmt.Sprintf("%s,pxmonitor.pxCentralEndpoint=%s,pxmonitor.oidcClientSecret=%s",
+														 monitorValues,
+														 endpoint,
+														 oidcSecret)
+
+				configMap, err = core.Instance().UpdateConfigMap(configMap)
+				Expect(err).NotTo(HaveOccurred())
+				logrus.Infof("Updated helm values config map for px-monitor: %s", configMap.Data[monitorApp])
+			})
+
+			Step("Install px-monitor", func() {
+				err := Inst().S.AddTasks(context, monitorOptions)
+				Expect(err).NotTo(HaveOccurred())
+
+				ValidateContext(context)
+				logrus.Infof("Successfully validated specs for px-monitor")
+			})
+		})
+
+		Step("Uninstall license server and monitoring", func() {
+			err := Inst().S.ScheduleUninstall(context, monitorOptions)
+			Expect(err).NotTo(HaveOccurred())
+			err = Inst().S.ScheduleUninstall(context, lsOptions)
+			Expect(err).NotTo(HaveOccurred())
+
+			ValidateContext(context)
+			logrus.Infof("Successfully uninstalled px-license-server and px-monitor")
+		})
+
+		Step("destroy apps", func() {
+			opts := make(map[string]bool)
+			opts[scheduler.OptionsWaitForResourceLeakCleanup] = true
+
+			TearDownContext(context, opts)
+			logrus.Infof("Successfully destroyed px-central")
+		})
+	})
+})
+
+var _ = AfterSuite(func() {
+	PerformSystemCheck()
+	ValidateCleanup()
+})
+
+func TestMain(m *testing.M) {
+	ParseFlags()
+	os.Exit(m.Run())
+}
```

Comment:
sorry, but I think the installation of Helm is not Torpedo responsibility. We use spawn to install any apps on environments. @harsh-px what are your thoughts?

Reply:
the goal is to use Torpedo to test helm install an application, the app installed through helm is treated the same as other apps like mysql, only difference is that it has a different AppSpec, which Torpedo would recognize and schedule a helm install for it.

Reply:
@mborodin-px if this was discussed offline, let's resolve the conversation with a comment here. Same for other open comments in the PR. Thanks!